src/app/app-platform/ContentAppPlatform.cpp

```diff
@@ -502,6 +502,12 @@ bool ContentAppPlatform::HasTargetContentApp(uint16_t vendorId, uint16_t product
         return true;
     }
 
+    if (!app->HasSupportedCluster(chip::app::Clusters::AccountLogin::Id))
```

Comment:
This file has `using namespace chip::app::Clusters;` so this can just be:
```suggestion
    if (!app->HasSupportedCluster(AccountLogin::Id))
```

Reply:
Updated!